build.savant

```diff
@@ -75,6 +76,8 @@ project(group: "org.primeframework", name: "prime-mvc", version: "4.22.7", licen
       dependency(id: "com.google.guava:guava:${guavaVersion}", skipCompatibilityCheck: true)
     }
     group(name: "test-compile", export: false) {
+      // for serialization in UserIDCookieSession
+      dependency(id: "com.inversoft:jackson5:${jackson5Version}")
```

Comment:
It may be worth justing a string for the data type for tests so we don't have to depend up this library even as a test dep?

Reply:
Fixed 5302acb85490077eb2052c615e1a1e24f5bb2641